tests/software_tests/can/packet/test_first_frame.py

```
@@ -0,0 +1,612 @@
+        self._patcher_get_max_sf_dl = patch(f"{SCRIPT_LOCATION}.get_max_sf_dl")
+        self.mock_get_max_sf_dl = self._patcher_get_max_sf_dl.start()
+
+    def teardown_method(self):
+        self._patcher_validate_raw_bytes.stop()
+        self._patcher_can_addressing_information.stop()
+        self._patcher_dlc_handler.stop()
+        self._patcher_get_max_sf_dl.stop()
+
+    # is_single_frame
+
+    @pytest.mark.parametrize("addressing_format, raw_frame_data, ai_data_bytes_number, expected_output", [
```

Ruff (0.12.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)